### DISCUSSION TITLE
Menu Entry Swapper Configuration add coal bag options?

### DISCUSSION THREAD

**Discussion by illillusion** [runelite/runelite#15866] [Ideas]
options to fill/empty coal bag with shift click/default click in bank?

...there are like 30 blast furnace worlds right now, im sure it would get people some people to download it, if they havent already!

**Comment by SassLife** [comment]
they have this! I swapped mine when I use it at mlm. Just hold shift on the coal bag and you can swap it to empty..
not sure if that's what you are talking about but that's what I did
